assets/js/admin.js

```diff
@@ -178,3 +178,29 @@ jQuery(document).ready(function($) {
 		$('#in-' + taxonomy + '-' + id + ', #in-popular-' + taxonomy + '-' + id).prop( 'checked', c );
 	});
 });
+
+// Select all elements with the class "promote_job"
```

Comment:
I'm not sure I totally understand what you mean here - do you mind explaining a bit more? 😄

Reply:
Sorry! Sure!

I mean that we could separate a little more the classes and the functions of the generic part of the modal, and add with other classes and functions the specific details of the promote button.

An application example: if we decide later to create a confirmation modal for the job deletion, we could just use the same modal, applying the extra logic only for the deletion.

In practice, it would mean using a more generic class to the dialog, (maybe something like `wpjm-modal`?). And separate functions to open/close the modal (the generic part) and update the `href` button (the specific promote button part).

As a final result, we'd be able to easily create other modals.

I think it would also help with the deactivate modal logic.

Does it make sense to you?

Reply:
Makes sense!  Fixed in 11d6fd03e117f5b3ccc2d208b168c8308f2f6e6f